### PR TITLE
[api] Port /connector APIs to public

### PR DESCRIPTION
**Pull request by Harshg999**
## What changes were proposed in this pull request?

- Add /connector APIs to the public pool

## How was this patch tested?

- Tested manually

### CODE REVIEW

File: desktop/core/src/desktop/api_public_urls.py [hunk 1]
```diff
@@ -105,3 +105,16 @@
...
+  re_path(r'^connector/instances/?$', api_public.get_connectors_instances, name='connector_get_instances'),
+
+  re_path(r'^connector/instance/new/(?P<dialect>[\w\-]+)/(?P<interface>[\w\-]+)$', api_public.new_connector, name='connector_new'),
+  re_path(r'^connector/instance/get/(?P<id>\d+)$', api_public.get_connector, name='connector_get'),
+  re_path(r'^connector/instance/delete/?$', api_public.delete_connector, name='connector_delete'),
+  re_path(r'^connector/instance/update/?$', api_public.update_connector, name='connector_update'),
+  re_path(r'^connector/instance/test/?$', api_public.test_connector, name='connector_test'),
+
+  re_path(r'^connector/examples/install/?$', api_public.install_connector_examples, name='connector_install_examples'),
+]
```
Comment (agl29) [inline desktop/core/src/desktop/api_public_urls.py:120]: add the Enter in the end (line no 120).
And docs for APIs?

Comment (Harshg999) [reply desktop/core/src/desktop/api_public_urls.py:120]: Adding docs in a separate PR

**Review by romainr — APPROVED** [review]
Nice!

And docs?

**Comment by Harshg999** [comment]
> Nice!
> 
> And docs?

Added in new PR #2330

**Pull request merged**